### PR TITLE
Updated `framer-motion`

### PR DESCRIPTION
**Pull request by hirotomoyamada**
## Description

Updated dependencies.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by hirotomoyamada** [comment]
@108yen @kurakke @taroj1205 @hirayamahiroto 

Fixed tests.

File: packages/components/modal/src/dialog.tsx
Comment (108yen) [inline packages/components/modal/src/dialog.tsx:1]: Is `changeset` of `Modal` unnecessary?

**Review by 108yen — CHANGES_REQUESTED** [review]
Do not include multiple assertions inside `waitFor`. Please put only the first one inside `waitFor`.
If you want to make `getxxx` asynchronous, use `findxxx`.


Are you sure that `forwardRef` is imported from `react` and not `@yamada-ui/core`?

File: packages/components/modal/src/dialog.tsx
Comment (hirotomoyamada) [reply packages/components/modal/src/dialog.tsx:1]: This does not change the internal logic, it just resolves the TypeScript error. So there is no need for `changeset`.

**Comment by hirotomoyamada** [comment]
@108yen 

This time, the `forwardRef` will be imported from React. The reason is that we want to genericize the component type.

**Comment by hirotomoyamada** [comment]
@108yen 

Fixed tests.

**Review by 108yen — DISMISSED** [review]

**Review by kurakke — DISMISSED** [review]

**Review by taroj1205 — DISMISSED** [review]

**Review by kurakke — APPROVED** [review]

**Review by 108yen — APPROVED** [review]

**Pull request merged**